config/monitor_services.go

```diff
@@ -165,6 +166,21 @@ func (c *ServicesMonitorConfig) Validate() error {
 		c.Regexp = String("") // Finalize
 	}
 
+	// Check that names does not contain empty strings
+	if namesConfigured {
+		emptyName := false
+		for _, name := range c.Names {
+			if name == "" {
+				emptyName = true
+				break
+			}
+		}
+		if emptyName {
+			return fmt.Errorf("names field includes empty string(s). services " +
+				"names cannot be empty")
+		}
```

Comment:
It seems like the goal is to error if there are empty names, which could be returned at the first occurrence instead of explicitly breaking out of the for loop
```suggestion
		for _, name := range c.Names {
			if name == "" {
				return fmt.Errorf("names field includes empty string(s). services " +
				"names cannot be empty")
			}
		}
```

Reply:
Thanks for the suggestion! Saves a variable and much more compact